device-connectors/src/testflinger_device_connectors/devices/muxpi/muxpi.py

```diff
@@ -149,7 +149,9 @@ def _copy_to_control(self, local_file, remote_file):
 
     def provision(self):
         # If this is not a zapper, reboot before provisioning
-        if "zapper" not in self.config.get("control_switch_local_cmd", ""):
```

Comment:
We needed to revert this part for now, so you'll want to rebase on current main. This section will likely conflict and need to be removed since we need to re-implement reboot_control_host() a different way.

Reply:
Thanks, rebased my branch and removed that commit.